components/package.json

```diff
@@ -8,39 +8,37 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@revolist/vue3-datagrid": "^3.0.96",
+    "@revolist/vue3-datagrid": "^3.2.17",
     "bootstrap-icons": "^1.3.0",
     "chart.js": "^3.9.1",
-    "core-js": "^3.6.5",
+    "core-js": "^3.26.1",
     "date-fns": "^2.22.1",
-    "mitt": "^2.1.0",
-    "nanoid": "^3.2.0",
-    "vue": "^3.2.2"
+    "mitt": "^3.0.0",
+    "nanoid": "^4.0.0",
+    "vue": "^3.2.45"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.19",
-    "@typescript-eslint/eslint-plugin": "^2.33.0",
-    "@typescript-eslint/parser": "^2.33.0",
-    "@vue/cli-plugin-babel": "~4.5.13",
-    "@vue/cli-plugin-eslint": "~4.5.13",
-    "@vue/cli-plugin-typescript": "~4.5.13",
-    "@vue/cli-plugin-unit-jest": "~4.5.13",
-    "@vue/cli-service": "~4.5.13",
-    "@vue/compiler-sfc": "^3.0.0",
-    "@vue/eslint-config-prettier": "^6.0.0",
-    "@vue/eslint-config-typescript": "^5.0.2",
-    "@vue/test-utils": "^2.0.0-0",
-    "csstype": "^3.0.6",
-    "eslint": "^6.7.2",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-vue": "^7.5.0",
-    "prettier": "^1.19.1",
-    "sass": "^1.26.5",
-    "sass-loader": "^8.0.2",
-    "svg-sprite-loader": "^5.2.1",
-    "svg-transform-loader": "^2.0.13",
-    "svgo-loader": "^2.2.1",
-    "typescript": "~4.1.5",
-    "vue-jest": "^5.0.0-0"
+    "@types/jest": "^27.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
+    "@typescript-eslint/parser": "^5.4.0",
+    "@vue/cli-plugin-babel": "~5.0.8",
+    "@vue/cli-plugin-eslint": "~5.0.8",
+    "@vue/cli-plugin-typescript": "~5.0.8",
+    "@vue/cli-plugin-unit-jest": "~5.0.8",
+    "@vue/cli-service": "~5.0.8",
+    "@vue/eslint-config-typescript": "^9.1.0",
+    "@vue/test-utils": "^2.0.0",
+    "@vue/vue3-jest": "^27.0.0",
+    "babel-jest": "^27.0.6",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-vue": "^8.0.3",
+    "jest": "^27.1.0",
```

Comment:
Cannot update to `vue-jest 29` based on `jest 29` due to `@vue/cli-plugin-unit-jest` plugin for `vue-cli` is not supported yet. (most likely will not be supported ever as upstream development is now focused on vite)